lib/server.js

```diff
@@ -39,6 +40,8 @@ function parseData(data) {
   };
 }
 
+const stat_files = ['package.json', 'package-lock.json', 'yarn.lock'];
```

Comment:
please add `npm-shrinkwrap.json` here, which is identical to `package-lock.json` but also works on npm < 5

Reply:
Good point. Done.